tgui/packages/tgui/components/Tooltip.tsx

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable react/no-deprecated */
+// TODO: Rewrite as an FC, remove this lint disable - JLS 01/11/2023
```

Comment:
```suggestion
// TODO: Rewrite as an FC, remove this lint disable
```

This doesn't really help